src/ChargeDriftModels/ADLChargeDriftModel/ADLChargeDriftModel.jl

```diff
@@ -319,8 +319,7 @@ end
     AE, RE
 end
 
-
-@fastmath function getVe(fv::SVector{3, T}, cdm::ADLChargeDriftModel{T,M,N}, Emag_threshold::T = T(1e-5))::SVector{3, T} where {T <: SSDFloat, M, N}
+@fastmath function getVe(fv::SVector{3, T}, cdm::ADLChargeDriftModel{T,M,N}, current_pos::CartesianPoint{T} = zero(CartesianPoint{T}), Emag_threshold::T = T(1e-5))::SVector{3, T} where {T <: SSDFloat, M, N}
```

Comment:
Here, I would still split `getVe` into two methods (the old one remaining ontouched, and adding a new one with `current_pos`:
```suggestion
getVe(fv::SVector{3, T}, cdm::ADLChargeDriftModel{T}, ::CartesianPoint{T}, Emag_threshold::T = T(1e-5)) where {T} = getVe(fv, cdm, Emag_threshold)

@fastmath function getVe(fv::SVector{3, T}, cdm::ADLChargeDriftModel{T,M,N}, Emag_threshold::T = T(1e-5))::SVector{3, T} where {T <: SSDFloat, M, N}
```